verl/utils/reward_score/__init__.py

```diff
@@ -14,25 +14,25 @@
 # from . import gsm8k, math, prime_math, prime_code
 
 
-def _default_compute_score(data_source, solution_str, ground_truth, extra_info=None):
+def _default_compute_score(data_source, prompt_str, response_str, ground_truth, extra_info=None):
```

Comment:
Same here. Shall we only pass the response_str? I guess this will break the PRIME reward manager.